flang/lib/Lower/OpenACC.cpp

```diff
@@ -516,6 +528,129 @@ genACC(Fortran::lower::AbstractConverter &converter,
   }
 }
 
+static void
+genACCExitDataOp(Fortran::lower::AbstractConverter &converter,
+                 const Fortran::parser::AccClauseList &accClauseList) {
+  mlir::Value ifCond;
+  SmallVector<Value, 2> copyoutOperands, deleteOperands, detachOperands,
+      waitOperands;
+  Value async, waitDevnum;
```

Comment:
`mlir::Value`? There is an `llvm::Value` class as well, so being precise here might save some grief later.

Reply:
Sure. I'll update that.